tests/test_a2a_boot.py

```
@@ -253,6 +259,69 @@ def test_weather_agent_card_advertises_the_oidc_security_scheme():
+
+    assert "ptso_agent" in module.expected_agent_names
+    assert "ptso_agent" in mounted_a2a_agent_names(module.app)
+
+    client = TestClient(module.app)
+    resp = client.get("/a2a/ptso_agent/.well-known/agent-card.json")
+    assert resp.status_code == 200
+    assert resp.json()["name"] == "ptso_agent"
+
+    api_server = _recover_api_server(module.app)
+
+    async def _served_root_shape():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_served_root_shape`

(ANN202)